### DISCUSSION TITLE
清爽首页功能

### DISCUSSION THREAD

**Discussion by Cat0x1** [the1812/Bilibili-Evolved#3357] [功能建议 / Ideas]
如果可以把简化首页加入清爽首页可以嘛？我喜欢清爽首页，但是也需要简化首页来屏蔽我不喜欢的栏目，而开启清爽首页后就无法使用简化首页的功能了
![image](https://user-images.githubusercontent.com/58590403/170217303-73d81453-11da-4cf0-8d62-365d210b133e.png)
![image](https://user-images.githubusercontent.com/58590403/170217405-b31dab42-b6bf-4d69-bf33-27322780317d.png)

**Comment by Geeyun-JY3** [comment]
准确的说，[简化首页](https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#简化首页) 是针对原版首页的功能。你提的这个应该算 [清爽首页](https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#清爽首页) 的进一步自定义选项。

**Reply by Cat0x1** [reply]
是的

**Comment by the1812** [comment]
好像以前也有人提过, 清爽首页因为一次只显示一个分区, 所以没啥好屏蔽的, 只要把自己喜欢的分区排序到最前面就行了

**Reply by Cat0x1** [reply]
就是不舒服这样看着。。。哈哈
